rust/actyx/ax-core/src/node/host.rs

```diff
@@ -83,6 +73,29 @@ impl Host {
     }
 }
 
+fn initialize_node_storage(base_path: &Path) -> Result<NodeStorage> {
+    Ok(if cfg!(test) {
+        NodeStorage::in_memory()
+    } else {
+        NodeStorage::new(base_path.join("node.sqlite")).context("Error opening node.sqlite")?
```

Comment:
Any chance we can use the full path here? It would avoid the user thinking "where though?"

Reply:
This should make it clearer
```suggestion
        let node_path = base_path.join("node.sqlite");
        NodeStorage::new(node_path).context(format!("Error opening {}", node_path.display()))?
```